api_auth.go

```diff
@@ -66,36 +67,44 @@ func (bot *Bot) StopHeartbeat() {
 }
 
 // StartHeartbeat renew auth token periodically
-func (bot *Bot) StartHeartbeat() {
+func (bot *Bot) StartHeartbeat() error {
 	if !bot.requireType(ChatBot) {
-		bot.logger.Log(bot.ctx, LogLevelError, "Heartbeat only support Chat Bot")
-		return
+		return fmt.Errorf("Heartbeat only support Chat Bot")
```

Comment:
There is an `ErrBotTypeError` for this scenario.

Reply:
Updated